scalafix/rules/src/main/scala/fix/v1.scala

```diff
@@ -530,13 +530,20 @@ object SinkToPipeRules {
   ).asPatch
 
   def apply(t: Tree)(implicit doc: SemanticDocument): List[Patch] = {
+    //println("Tree.structureLabeled: " + doc.tree.structureLabeled)
+    //val sinkMatcher = SymbolMatcher.normalized("fs2.Sink")
+    val sinkMatcher = SymbolMatcher.exact("fs2/Sink.")
```

Comment:
What happens with `fs2/Sink#`?

Reply:
It doesn't match but I'm not sure why.

Reply:
What symbol is printed by `Tree.symbol` of the trees you want to match?

Reply:
BTW, what's the difference between `fs2/Sink.` and `fs2/Sink#`?  I don't think the docs are clear on that.

Reply:
the symbol format is documented in the SemanticDB specification https://scalameta.org/docs/semanticdb/specification.html#symbol

`Sink.` is the object symbol and `Sink#` is the class or trait symbol